### PR TITLE
feat: add fec profiles functionality - initial setup 

### PR DESCRIPTION
**Pull request by mobregozo**
### Context

Profiles FEC Initial version.
Find more info about the details of this feature in [here](https://docs.google.com/document/d/1b-j2TRWopN12yQBXPd3Lv6jCHKlgxN9NSkYP_TjRx5E/edit)

[Perfiles Listado](https://website-git-feature-profiles-setup-hifrontendcafe.vercel.app/perfiles)
[Demo Perfil](https://website-git-feature-profiles-setup-hifrontendcafe.vercel.app/perfiles/manu-obregozo)

This enables the possibility to generate and load profiles pages statically by including `.md` files under the `/_profiles` folder.

### CODE REVIEW

File: src/components/ProfileSocialMedia/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                height={24}
+              />
+            </a>
+          ),
+      )}
+    </div>
+  );
+};
+
+export default ProfileSocialMedia;
```
Comment (marbiano) [inline src/components/ProfileSocialMedia/index.tsx:42]: I saw an initial glitch on the social media icons for the profile link you shared, do you know why is it happening? Could it be related to this component?

**Review by marbiano — CHANGES_REQUESTED** [review]
Amazing work! Some minor comments, but in general this PR looks really clean, and the feature came along quite nicely. Thank you, @mobregozo!

File: src/_profiles/manu-obregozo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: 'Manuel Obregozo'
+discord: 'Manu O.#1258'
+openToWork: true
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/_profiles/manu-obregozo.md:4]: Done!

File: src/components/ProfileBody/index.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import markdownStyles from './styles.module.css';
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileBody/index.tsx:1]: sure, not preferences in here I took this approach from the example Vercel has in [here](https://github.com/vercel/next.js/blob/canary/examples/blog-starter-typescript/components/post-body.tsx)

File: src/components/ProfileBody/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  @apply my-6;
+}
+
+.markdown h2 {
+  @apply text-3xl mt-12 mb-4 leading-snug;
+}
+
+.markdown h3 {
+  @apply text-2xl mt-8 mb-4 leading-snug;
+}
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileBody/styles.module.css:18]: No, we are not covering many scenarios with the ones I have added. 
It was just to cover the simple cases. Should I add more or can we iterate on this in another ticket?

File: src/components/ProfileCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    <Link href={`/perfiles/${profile.slug}`}>
+      <a>
+        <div className="p-2 py-4 shadow-md rounded-lg text-center border-b-4 transform hover:scale-105 transition-transform duration-200">
+          <img
+            src={profile.coverImage}
+            className="mx-auto h-24 w-24 object-cover object-top rounded-full border"
+            alt={profile.name}
+          />
+          {profile.openToWork && (
+            <div className="p-2 font-semibold text-primary text-center ">
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileCard/index.tsx]: Fixed!

File: src/components/ProfileCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+              En búsqueda activa
+            </div>
+          )}
+          <h1 className="text-lg md:text-xl font-bold tracking-tighter leading-tight md:leading-none">
+            {profile.name}
+          </h1>
+          <h2 className="font-bold text-gray-500 tracking-wide">
+            {profile.role}
+          </h2>
+          {profile.stack && profile.stack.length > 0 && (
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileCard/index.tsx:29]: makes sense, I am still not familiar with the optional chaining. Good catch!

File: src/components/ProfileCard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+            <div className="flex items-center flex-wrap justify-center">
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileCard/index.tsx]: Done!

File: src/components/ProfileBody/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileBody/index.tsx]: done!

File: src/components/ProfileBody/index.tsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+type Props = {
+  content: string
+}
+
+const ProfileBody = ({ content }: Props) => {
+  return (
+    <div className="mx-auto">
+      <div
+        className={markdownStyles['markdown']}
```
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileBody/index.tsx]: Done.

File: src/components/ProfileBody/styles.module.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (marbiano) [reply src/components/ProfileBody/styles.module.css:18]: Maybe we can add some basic ones for `ul` and `ol`, and iterate on the rest as profiles start getting added?

File: src/components/ProfileSocialMedia/index.tsx (hunk 1 above)
Comment (marbiano) [reply src/components/ProfileSocialMedia/index.tsx:42]: Ah, gotcha. Yeah, seems like a good idea, altough I think we might want to reinforce adding proper images while reviewing new profiles, which would make this case impossible to happen. But when the review is manual, then there's nothing impossible, so I like the idea of having a fallback for it.

There might be a simpler way to handle this, but for now I think it's a great start.

File: src/components/ProfileBody/styles.module.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (mobregozo) [reply src/components/ProfileBody/styles.module.css:18]: Just found this, which solves our problem: https://github.com/tailwindlabs/tailwindcss-typography 🤯

File: src/components/ProfileBody/index.tsx [hunk 9]
```diff
@@ -1,18 +1,16 @@
...
+  content: string;
+};
 
 const ProfileBody = ({ content }: Props) => {
   return (
     <div className="mx-auto">
-      <div
-        className={markdownStyles['markdown']}
+      <article
+        className="prose prose-xl"
```
Comment (marbiano) [inline src/components/ProfileBody/index.tsx:9]: Good one! 👌

**Review by marbiano — APPROVED** [review]
LGTM, great work! 🚀

**Pull request merged**